tools/tests/ANSI Test.ipynb

```diff
@@ -34,6 +34,42 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "# Bold, underline and inverse text"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "This is normal text\n",
+      "\n",
+      "\u001b[01mThis is bold text\n",
+      "\n",
+      "\u001b[04mThis is underlined text\n",
+      "\n",
+      "\u001b[07mThis is inverse text\n"
+     ]
+    }
+   ],
+   "source": [
+    "print (\"This is normal text\")\n",
+    "print()\n",
+    "print (\"{ESC}01mThis is bold text\".format(**locals()))\n",
+    "print()\n",
+    "print (\"{ESC}04mThis is underlined text\".format(**locals()))\n",
+    "print()\n",
+    "print (\"{ESC}07mThis is inverse text\".format(**locals()))"
```

Comment:
Is it intentional that you don't reset the style between lines, so this actually generates:
- normal
- bold
- bold + underlined
- bold + underlined + inverse